hip-service/wwwroot/swagger.yaml

```diff
@@ -0,0 +1,172 @@
+      - application/xml
+      requestBody:
+        content:
+          application/json:
+            required: true
+            schema:
+              $ref: '#/components/requestBodies/PatientDiscoveryRequest'
+          application/xml:
+            required: true
+            schema:
+              $ref: '#/components/requestBodies/PatientDiscoveryRequest'
+      responses:
+        '200':
+          description: A patient record with one or more care contexts
+          schema:
+            $ref: '#/components/responses/PatientDiscoveryResponse'
+        '400':
+          description: >
+            **Causes:**
+              * Empty verified identifiers.
+              * Format mismatch of any of attributes.
+                | type   | Format/Allowed Values|
+                | ------- | ----------------    |
+                | dateOfBirth     | dd-mm-yyyy          |
+                | gender  | male/female/unknown |
+                | phone-number| 10 digits       |
+
+          schema:
+            $ref: '#/components/responses/PatientDiscoveryErrResponse'
+        '404':
+          description: >
+            **Causes:**
+              * No matching patient record found for the given identifiers.
+              * There are more than one definitive match for a given request.
+          schema:
+            $ref: '#/components/responses/PatientDiscoveryErrResponse'
+        '500':
+          description: >
+            **Causes:**
+              * Downstream system(s) is down.
+              * Unhandled exceptions.
+          schema:
+            $ref: '#/components/responses/PatientDiscoveryErrResponse'
+
+components:
+  requestBodies:
+    PatientDiscoveryRequest:
+      type: object
+      properties:
+        verifiedIdentifiers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Identifier'
+          xml:
+            name: verifiedIdentifiers
+            wrapped: true
+        unverifiedIdentifiers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Identifier'
+          xml:
+            name: unverifiedIdentifiers
+            wrapped: true
+        firstName:
+          type: string
+          example: "chandler"
+        lastName:
+          type: string
+          example: "bing"
+        gender:
+          type: gender
+          enum: [male, female, unknown]
+        dateOfBirth:
+          type: string
+          format: date
+      xml:
+        name: PatientDiscoveryRequest
+
+  responses:
+    PatientDiscoveryResponse:
+      type: object
+      properties:
+        patient:
+            $ref: '#/components/schemas/Patient'
+      xml:
+        name: PatientDiscoveryResponse
+
+    PatientDiscoveryErrResponse:
+      type: object
+      properties:
+        error:
+          $ref: '#/components/schemas/Error'
+
+  schemas:
+    Identifier:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [phone-number, case-id]
```

Comment:
- Ideally we should follow some definitive set/examples. Not sure if MDDS has any. 
- FHIR identifier types are here - https://www.hl7.org/fhir/valueset-identifier-type.html
- IMO, enums can be - AADHAAR, MOBILE, EPIC, PAN, BRN, DL, PPN, MR etc. We can use with MR, MOBILE to start with.

Reply:
```suggestion
          enum: [MR, MOBILE]
```